FindMe/View Controllers/FindMeViewController.m

```diff
@@ -60,13 +62,21 @@ - (IBAction)onFindMe:(id)sender
     }
     else {
         [self designTrackingButtonWithState:@"unselected"];
-        [self.locationManager stopTracking];
+        [self.locationManager stopTracking:self.dateAndTimeNow];
```

Comment:
The trace objects should own the stop time. Should be able to call `[self.trace stop]` and it should be able to set the stop date on itself.

Reply:
I have created a trace object to take care of everything trace related. I should have done that the first time. Thanks for the feedback😊